### ISSUE TITLE
Dynamc child pipelines are not visible

### ISSUE THREAD

**Issue by kivipe** [heikkipora/gitlab-radiator#93]
I have a pipeline that runs a child job with trigger:influence:artifact. Radiator shows only triggering part of the pipeline and not triggered part.

**Comment by heikkipora** [comment]
Would you like to provide a small sample pipeline YML file that can be used to reproduce that situation?

**Comment by kivipe** [comment]
Hi, here are two files:

.gitlab-ci.yml:
```
stages:
   - start
   - run-child

start-job:
  stage: start
  script:
    - echo start stage

run-child:
  stage: run-child
  trigger:
    include:
      - .gitlab-child.yml
    strategy: depend
```

.gitlab-child.yml
```
stages:
  - build
  - test

build:
  stage: build
  script:
  - echo build stage

test:
  stage: test
  script:
  - echo test stage
```

**Comment by heikkipora** [comment]
Interesting - I was not aware that one can include a "child pipeline declaration" like that. 
I'm going to investigate how that manifests through the GitLab API in the near future, stay tuned

**Comment by heikkipora** [comment]
Fortunately GitLab had a separate API call for getting the relationships ("bridge jobs" they call them) between parent and child pipelines.

A rather straightforward implementation produces now this based on your example pipeline declaration:

<img width="743" alt="image" src="https://user-images.githubusercontent.com/259924/191338458-be132170-75f0-4442-a73f-cfe68b2c8a90.png">

What do you think?

**Comment by kivipe** [comment]
That looks great!

**Comment by heikkipora** [comment]
Released as v3.4.0 (https://www.npmjs.com/package/gitlab-radiator/v/3.4.0)
